Header header
int16 accelX
int16 accelY
int16 accelZ
int16 gyroX
int16 gyroY
int16 gyroZ
int16 magX
int16 magY
int16 magZ
float32 pitch
float32 roll
float32 yaw	#yaw = heading if yaw used
uint32 time
